src/functions/module_financial.ts

```diff
@@ -890,6 +890,38 @@ export const FV: AddFunctionDescription = {
   },
 };
 
+// -----------------------------------------------------------------------------
+// FVSCHEDULE
+// -----------------------------------------------------------------------------
+export const FVSCHEDULE: AddFunctionDescription = {
+  description: _lt("Future value of principal from series of rates."),
+  args: args(`
+  principal (number) ${_lt("The amount of initial capital or value to compound against.")}
+  rate_schedule (number, range<number>) ${_lt(
+    "A series of interest rates to compound against the principal."
+  )}
+  `),
+  returns: ["NUMBER"],
+  compute: function (
+    principalAmount: PrimitiveArgValue,
+    rateSchedule: MatrixArgValue | PrimitiveArgValue
+  ): number {
+    const principal = toNumber(principalAmount);
+    if (!Array.isArray(rateSchedule)) {
```

Comment:
Cannot use `visitNumbers` since we want to throw an error when encountering a value that cannot be coerced to a number, but using `reduceAny`works well :)

Reply:
let's go for ```reduceAny```